src/ui/controls/color-dropdown/index.tsx

```diff
@@ -12,6 +12,8 @@ interface ColorDropDownProps {
     hasAutoOption?: boolean;
     onChange: (value: string) => void;
     onReset: () => void;
+    cssValue: string;
+    cssChange: (cssValue: string) => void;
```

Comment:
I don't think we should have a separate function for the variables one. It can reuse the `onChange` function. It does needs some logic to change the value for the correct property.